api/envoy/extensions/filters/http/admission_control/v3alpha/admission_control.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.admission_control.v3alpha;
+
+import "envoy/config/core/v3/base.proto";
+import "envoy/type/v3/range.proto";
+
+import "google/api/annotations.proto";
+import "google/protobuf/duration.proto";
+import "google/protobuf/wrappers.proto";
+import "google/rpc/status.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.admission_control.v3alpha";
+option java_outer_classname = "AdmissionControlProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Admission Control]
+// [#extension: envoy.filters.http.admission_control]
+
+message AdmissionControl {
+  // Default method of specifying what constitutes a failed request. All status codes that
+  // indicate a failed request must be explicitly specified if not relying on the default
+  // values.
+  message DefaultEvaluationCriteria {
+    // If HTTP statuses are unspecified, defaults to 5xx.
```

Comment:
Can you comment on valid ranges here and make sure we have code checks for them?